src/PowerShellEditorServices/Services/PowerShell/Handlers/GetCommentHelpHandler.cs

```diff
@@ -47,14 +38,14 @@ public async Task<CommentHelpRequestResult> Handle(CommentHelpRequestParams requ
                 triggerLine,
                 out string helpLocation);
 
-            if (functionDefinitionAst == null)
+            if (functionDefinitionAst is null)
             {
                 return result;
             }
 
             IScriptExtent funcExtent = functionDefinitionAst.Extent;
             string funcText = funcExtent.Text;
-            if (helpLocation.Equals("begin"))
+            if (helpLocation.Equals("begin", StringComparison.Ordinal))
```

Comment:
consider:

```suggestion
            if (helpLocation is "begin")
```

I like to do this if I'm doing ordinal. Dunno if roslynator complains about not explicit intention though.